tests/cli/workflow/rnafusion/test_cli_rnafusion_compound_commands.py

```diff
@@ -123,47 +147,18 @@ def test_store_fail(
     assert result_fail.exit_code != EXIT_SUCCESS
 
 
-def test_start_available(
-    cli_runner: CliRunner,
-    rnafusion_context: CGConfig,
-    caplog: LogCaptureFixture,
-    mocker,
-    rnafusion_case_id: str,
-):
-    """Test to ensure all parts of compound start-available command are executed given ideal conditions
-    Test that start-available picks up eligible cases and does not pick up ineligible ones."""
-    caplog.set_level(logging.INFO)
-
-    # GIVEN CASE ID of sample where read counts pass threshold
-    case_id_success: str = rnafusion_case_id
-
-    # GIVEN a mocked config
-
-    # GIVEN decompression is not needed
-    mocker.patch.object(RnafusionAnalysisAPI, "resolve_decompression")
-    RnafusionAnalysisAPI.resolve_decompression.return_value = None
-
-    # WHEN running command
-    result = cli_runner.invoke(start_available, ["--dry-run"], obj=rnafusion_context)
-
-    # THEN command exits with 0
-    assert result.exit_code == EXIT_SUCCESS
-
-    # THEN it should successfully identify the one case eligible for auto-start
-    assert case_id_success in caplog.text
-
-
 def test_store_available(
     cli_runner: CliRunner,
     rnafusion_context: CGConfig,
     real_housekeeper_api,
-    mock_deliverable,
-    mock_analysis_finish,
-    caplog: LogCaptureFixture,
-    mocker,
     hermes_deliverables,
     rnafusion_case_id: str,
+    deliverables_template_content: List[dict],
+    mock_deliverable,
+    mock_analysis_finish,
```

Comment:
type hints

Reply:
These are functions that do a bunch of stuff (like creating files and folders) but return None. I could add Callable as type hint but not sure if that really helps

Reply:
Oh, I thought it was a bit unclear what they returned, but then it makes sense to not have them type hinted haha. I will resolve this.